Adding PR- and Issue-templates (#228)
NVIDIA-Merlin/Transformers4Rec: CONTRIBUTING.md

# Contributing to Transformers4Rec

> If you are interested in contributing to NVTabular your contributions will fall

Reviewer: ```suggestion
If you are interested in contributing to Transformers4Rec your contributions will fall
```

into three categories:
1. You want to report a bug, feature request, or documentation issue
    - File an [issue](https://github.com/NVIDIA-Merlin/Transformers4Rec/issues/new/choose)
    describing what you encountered or what you want to see changed.

>     - The NVTabular team will evaluate the issues and triage them, scheduling

Reviewer: ```suggestion
    - The NVIDIA-Merlin team will evaluate the issues and triage them, scheduling
```

    or [help wanted](https://github.com/NVIDIA-Merlin/Transformers4Rec/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) labels
3. Comment on the issue saying you are going to work on it
4. Code! Make sure to update unit tests!
5. When done, [create your pull request](https://github.com/NVIDIA-Merlin/Transformers4Rec/compare)
6. Verify that CI passes all [status checks](https://help.github.com/articles/about-status-checks/). Fix if needed
7. Wait for other developers to review your code and update code as needed
8. Once reviewed and approved, a developer will merge your pull request

Remember, if you are unsure about anything, don't hesitate to comment on issues
and ask for clarifications!

### Seasoned developers

Once you have gotten your feet wet and are more comfortable with the code, you
can look at the prioritized issues of our next release in our [project boards](https://github.com/NVIDIA-Merlin/Transformers4Rec/projects).

> **Pro Tip:** Always look at the release board with the highest number for

> issues to work on. This is where NVTabular developers also focus their efforts.

Reviewer: ```suggestion
issues to work on. This is where Transformers4Rec developers also focus their efforts.
```
